upb/table.int.h

```diff
@@ -63,10 +63,14 @@ typedef struct {
 #endif
 
 /* Like strdup(), which isn't always available since it's not ANSI C. */
-char *upb_strdup(const char *s);
+char *upb_strdup(const char *s, upb_alloc *a);
```

Comment:
this seems like an odd file for this, it's used by other things than just table now, right?